File: sdkwork-rtc-sdk-kotlin/providers/rtc-sdk-provider-jitsi/src/main/kotlin/com/sdkwork/rtc/provider/jitsi/RtcProviderJitsiPackageContract.kt
package com.sdkwork.rtc.provider.jitsi

/**
 * Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
 */
object RtcProviderJitsiPackageContract {
    const val PROVIDER_KEY: String = "jitsi"
    const val PLUGIN_ID: String = "rtc-jitsi"
    const val DRIVER_ID: String = "sdkwork-rtc-driver-jitsi"
    const val PACKAGE_IDENTITY: String = "com.sdkwork:rtc-sdk-provider-jitsi"
    const val STATUS: String = "future-runtime-bridge-only"
    const val RUNTIME_BRIDGE_STATUS: String = "reserved"
    const val ROOT_PUBLIC: Boolean = false
}

File: sdkwork-rtc-sdk-csharp/providers/Sdkwork.Rtc.Sdk.Provider.Jitsi/src/RtcProviderJitsiPackageContract.cs
namespace Sdkwork.Rtc.Sdk.Provider.Jitsi;

/// <summary>
/// Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
/// </summary>
public static class RtcProviderJitsiPackageContract
{
    public const string ProviderKey = "jitsi";
    public const string PluginId = "rtc-jitsi";
    public const string DriverId = "sdkwork-rtc-driver-jitsi";
    public const string PackageIdentity = "Sdkwork.Rtc.Sdk.Provider.Jitsi";
    public const string Status = "future-runtime-bridge-only";
    public const string RuntimeBridgeStatus = "reserved";
    public const bool RootPublic = false;
}
